src/Functions/FunctionsBitmap.cpp

```diff
@@ -9,12 +9,188 @@ namespace DB
+        R"(
+┌─res───┐
+│ [5,3] │
+└───────┘
+    )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_bitmapSubsetLimit = {20, 1};
+    FunctionDocumentation::Category category_bitmapSubsetLimit = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_bitmapSubsetLimit = {
+        description_bitmapSubsetLimit,
+        syntax_bitmapSubsetLimit,
+        arguments_bitmapSubsetLimit,
+        returned_value_bitmapSubsetLimit,
+        examples_bitmapSubsetLimit,
+        introduced_in_bitmapSubsetLimit,
+        category_bitmapSubsetLimit
+    };
+
+    factory.registerFunction<FunctionBitmapSubsetLimit>(documentation_bitmapSubsetLimit);
+
+    // Documentation for subBitmap (bitmapSubsetOffsetLimit is the unregistered alias see: https://github.com/ClickHouse/ClickHouse/pull/27234)
+    FunctionDocumentation::Description description_subBitmap = "Returns a subset of the bitmap, starting from position `offset`. The maximum cardinality of the returned bitmap is `cardinality_limit`.";
+    FunctionDocumentation::Syntax syntax_subBitmap = "subBitmap(bitmap, offset, cardinality_limit)";
+    FunctionDocumentation::Arguments arguments_subBitmap = {
+        {"bitmap", "Bitmap object. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)."},
+        {"offset", "Number of elements to skip from the beginning (zero-based). [`UInt32`](/sql-reference/data-types/int-uint)"},
+        {"cardinality_limit", "Maximum number of elements to include in the subset. [`UInt32`](/sql-reference/data-types/int-uint)"}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_subBitmap = "Returns a bitmap containing at most `limit` elements, starting after skipping `offset` elements in ascending order. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)";
+    FunctionDocumentation::Examples examples_subBitmap = {{"Usage example", "SELECT bitmapToArray(subBitmap(bitmapBuild([1, 2, 3, 4, 5]), 2, 2)) AS res;",
+        R"(
+┌─res───┐
+│ [3,4] │
+└───────┘
+    )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_subBitmap = {21, 9};
+    FunctionDocumentation::Category category_subBitmap = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_subBitmap = {
+        description_subBitmap,
+        syntax_subBitmap,
+        arguments_subBitmap,
+        returned_value_subBitmap,
+        examples_subBitmap,
+        introduced_in_subBitmap,
+        category_subBitmap
+    };
+
+    factory.registerFunction<FunctionBitmapSubsetOffsetLimit>(documentation_subBitmap);
+
+    // Documentation for bitmapTransform
+    FunctionDocumentation::Description description_bitmapTransform = R"(
+Replaces at most N bits in a bitmap. The old and new value of the i-th replaced bit is given by `from_array[i]` and `to_array[i]`.
```

Comment:
I'm afraid few users will understand what l. 160-162 mean. Rewrite in a more user friendly way?